arangopipe/DockerfileManagedService

```diff
@@ -0,0 +1,37 @@
+# This image uses two base images arangodb (above) and tensorflow (below)
```

Comment:
Why do we need our own docker image for the managed service?

Reply:
@joerg84 , a case for providing one is to provide a recipe for using arangopipe in this manner. If we don't provide one, then:

1.  We don't have a way to download and see all the bells and whistles easily. Somebody wanting to take a look will have to go through the hassle of installing all the peices in order. Install arangoDB first, then install arangopipe and then load the data, install the front end, etc.
2. We will have to figure out a way of decoupling the peices and still provide a quick user view of all the functionality. Not saying it is not possible or is possible - just that we have to figure out a way to provide a good experience for someone who wants to look at all the moving parts together.

Reply:
Why can't we use the normal docker image?

Reply:
@joerg84 , I don't understand the question. The image has arangodb, node and tensorflow and each of them provide a base docker image in their docker file recipe. This image requires all three.

Reply:
@joerg84, I think one solution for a docker image with just the front end, tensorflow (or torch) and the arangopipe api would be to use a simple python script at container startup that calls the foxx api on the remote machine and obtains the database name and password. This name and password would then be applied to the references to the host connection, which are:
(1) A config file for the front end
(2) Test cases for CI and container
(3) Test data generator and loader
Once the host connection has been applied to these references, we should be ok. Let me know if this seems like a better solution.

Reply:
As discussed the optimal solution are 3 deployment modes:
1. docker image bundled with arango for demo
2. standalone docker for prod deployments
3. Notebook only deployment (e.g., against managed service)

Reply:
@joerg84 . For the standalone docker for prod deployments, the image could contain just:
1. arango python dependencies (python-arango driver and arangopipe)
2. Jupyter notebook

Reply:
@joerg84 I have added the standalone docker file. I have a test script added to the docker image to test:
 (1) Arangopipe install 
(2) Connectivity to a server

Reply:
@joerg84 I submitted a request from google colab to link the example notebook (for (3)) to this repo.

Reply:
@joerg84 Updated the README file with guidelines on picking an Arangopipe version and added the colab notebook that demonstrates using arangopipe with a managed services database.